package.json

```diff
@@ -23,6 +23,7 @@
   },
   "packageManager": "yarn@3.6.3",
   "resolutions": {
-    "nomnom": "^1.8.1" 
+    "nomnom": "^1.8.1",
+    "autoprefixer": "10.4.5"
```

Comment:
Could you please explain the reason for the resolution of autoprefixer?
- why do we need to mention it here,
- why don't you use `^` (caret),
- doesn't it contradict `"^10.4.17"` in the lib package?

Reply:
The auroprefixer's authors made the breaking change in the patch version. So the main advice is to fix the version be equal to 10.4.5 for now. 
https://github.com/twbs/bootstrap/issues/36259